gsoc/making-patches/index.md

```diff
@@ -10,11 +10,10 @@ see that you can, indeed, succeed as you will need to be able to work on
 wxWidgets sources during your GSoC project.
 
 To get a general understanding of how submitting patches works, please read
-our [patches HOWTO](https://trac.wxwidgets.org/wiki/HowToSubmitPatches). Next,
-you need to select a task to work on. While you could take any open ticket
-in our [Trac](https://trac.wxwidgets.org) installation, we have a special list
-of [simple bugs](https://trac.wxwidgets.org/query?status=accepted&status=confirmed&status=infoneeded&status=infoneeded_new&status=new&status=reopened&keywords=~simple) which are
-supposed to be easy to fix so we recommend that you look at one of those.
+our [patches HOWTO](/develop/how-to-submit-patches/). Next,
+you need to select a task to work on. Please search for open tickets containing
```

Comment:
```suggestion
you need to select a task to work on. Please search for open issues containing
```
Searching for "simple" is not that simple. On wxTrac, it was added to keywords, but it seems this keyword was not translated to a issue label. But there probably is not much to be done about it.

Reply:
Yes, I realized this when it was too late already, i.e. while the import was running.